hs_err_pid716.log

```diff
@@ -0,0 +1,7723 @@
+#
```

Comment:
I'm still not sure why Brett added this to our source, but I guess we have to roll with it because it's in QA now, and that's what was tested (same comment for cantaloupe.properties.old)